.github/CODEOWNERS

```diff
@@ -75,3 +75,10 @@ subprojects/docs/src/docs/userguide/dep-man/
 subprojects/docs/src/docs/userguide/jvm/                                          @gradle/bt-jvm
 subprojects/docs/src/docs/userguide/reference/directory_layout.adoc               @gradle/bt-jvm
 subprojects/docs/src/docs/userguide/troubleshooting/version_catalog_problems.adoc @gradle/bt-jvm
+
+# Dev-prod team
+build-logic/                                @gradle/bt-developer-productivity
+build-logic-commons/                        @gradle/bt-developer-productivity
+build-logic-settings/                       @gradle/bt-developer-productivity
+subprojects/*/build.gradle*                 @gradle/bt-developer-productivity
+*.gradle.kts                                @gradle/bt-developer-productivity
```

Comment:
🤔 The `.teamcity` project should be added here. I wonder about `.github/workflows` as well.

Reply:
Added:

```
.teamcity/                                  @gradle/bt-developer-productivity
.github/                                    @gradle/bt-developer-productivity
gradle/shared-with-buildSrc                 @gradle/bt-developer-productivity
```